.github/workflows/web.yaml

```diff
@@ -0,0 +1,48 @@
+# Separate workflow for the wasmer-web crate.
+#
+
+# Automatically cancel previous workflow runs when a new commit is pushed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+name: Wasmer Web
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  web:
+    name: Build and Test (wasmer-web)
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [nightly]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install Rust
+      uses: actions/checkout@v3
+      with:
+        toolchain: ${{ matrix.rust }}
+
+    - name: Show Rust version
+      run: |
+        rustc --version
+
+    - name: Check formatting
```

Comment:
Is formatting already covered by the regular formatting check?
Or is that also just run for specific crates?

Reply:
Whoops, wrong PR.